Deploy/stacks/dynamic/stack-data-uploader/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 services:
   stack-data-uploader:
-    image: ghcr.io/cambridge-cares/stack-data-uploader${IMAGE_SUFFIX}:1.24.0
+    image: ghcr.io/cambridge-cares/stack-data-uploader${IMAGE_SUFFIX}:1.23.7
```

Comment:
Please revert the stack version numbers. Merging this into main will mess everything up